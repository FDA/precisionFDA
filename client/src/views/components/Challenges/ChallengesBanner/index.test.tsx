import React from 'react'
import { shallow } from 'enzyme'

import { ChallengesBanner } from '.'


describe('ChallengesBanner', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ChallengesBanner />)
    expect(wrapper).toMatchSnapshot()
  })
})
