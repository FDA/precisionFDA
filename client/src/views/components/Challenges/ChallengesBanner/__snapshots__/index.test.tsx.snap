// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChallengesBanner matches snapshot 1`] = `
<ChallengesBanner__StyledChallengesBanner>
  <ChallengesBanner__StyledChallengesBannerLeft>
    <h4>
      precisionFDA
    </h4>
    <h2>
      Challenges
    </h2>
  </ChallengesBanner__StyledChallengesBannerLeft>
  <ChallengesBanner__StyledChallengesBannerRight>
    <ViewAllButton
      title="View All Challenges"
      url="/challenges"
    />
  </ChallengesBanner__StyledChallengesBannerRight>
</ChallengesBanner__StyledChallengesBanner>
`;
